models/TradingAccount.py

```diff
@@ -411,7 +411,7 @@ def buy(self, cryptoMarket, fiatMarket, fiatAmount=0, manualPrice=0.00000000):
                         price = float(json['price'])
 
                 # calculate purchase fees
-                fee = fiatAmount * 0.005
+                fee = fiatAmount * getTakerFee()
```

Comment:
I don't know why, but only see people using `self.method()` in Python, maybe that's the convention?

Reply:
It's good to use that way especially if you need to distinguish calling member - non member functions having same name.